src/graphql/types/Query/organization.ts

```
@@ -64,3 +83,64 @@ builder.queryField("organization", (t) =>
 	}),
 );
+
+interface QueryCustomFieldsArgs {
+	input: {
+		organizationId: string;
+	};
+}
+
+interface ParsedArgs {
+	data?: QueryCustomFieldsArgs;
+	error?: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)